code-challenges.js

```diff
@@ -31,9 +93,33 @@ var hand3 = [5, 5, 5, 5, 4]
 var hand4 = [3, 5, 3, 5, 3]
 // Expected output: true
 
-
-// STRETCH:
+//A function taking an array of 5 numbers
+//if you .sort the array and make 5 numbers in numerical order
+// 6, 2, 4, 2, 2 >SORTED>>> 2, 2, 2, 4, 6
+//there are only two possible "winning" scenarios if your logic in comparing only 5 "cards"
+//if numbers start 0-4, if 0==1==2, and 3==4 or else if 0==1 and 2==3==4
+//true (You have a full house)
+//anything else would be false
+//A function taking an array of 5 numbers
+ const fullHouse = (array) => {
+//.sort least to greatest
+   let sortArray = array.sort((a,b)=>a-b)
+//if index 0 equals index 1, index 2.....
+   if (sortArray[0] === sortArray[1] && sortArray[0] === sortArray[2] && sortArray[3] === sortArray[4]){
```

Comment:
Consider using a data structure that models a full house. This explicit check does the trick but it's not obvious to someone else what this conditional is trying to do.